packages/action-bar/test/benchmark/basic-test-scoped.ts

```diff
@@ -0,0 +1,19 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import '@spectrum-web-components/action-bar/sp-action-bar-scoped.js';
```

Comment:
@Westbrook  It's one of the way we can add performance test, it will create a single table for entire action-bar but there is one issue here with the import as `@spectrum-web-components/action-bar/sp-action-bar-scoped.js` this is not getting resolved by the browser we might need to add support for such imports in the build system somewhere any suggestions how this import can be detected by browser?.

Reply:
On running `yarn test:bench --browser=firefox` locally I get this error in the test browser  `Uncaught (in promise) TypeError: Failed to resolve module specifier "@spectrum-web-components/action-bar/sp-action-bar-scoped.js". Relative references must start with either "/", "./", or "../".` I have already included this file in export.json & package.json but in the test browser it's not able to load However if I use relative import what the error suggest then I would run into other imports error which are using the package-name(aliasing) import format internally.

Reply:
`yarn test:bench --browser=firefox` will compare to the last published version, which does not have these files. 

Try `yarn test:bench --browser=firefox --compare=none` to not test against the previous release.

Reply:
Thanks it works. Locally below is the performance difference ::

Benchmark                                  │ Bytes            │ Avg time  │ vs action-bar:basic-test-scoped │ vs action-bar:basic-test
action-bar:basic-test-scoped   | 532.07 KiB   │ 92.86ms - 94.42ms │                                 │                   unsure │
action-bar:basic-test                 │ 472.93 KiB   │ 92.33ms - 93.91ms  │                    unsure │                                │

Reply:
Glad that it's running. The polyfill is still available in all versions of the code, so there's not a while lot of practical difference between the two versions right now...